controllers/tests/controller_server/host_definer/csi_node_watcher_test.py

```diff
@@ -1,322 +1,212 @@
-from unittest.mock import Mock
 from kubernetes.client.rest import ApiException
 
-import controllers.tests.controller_server.host_definer.utils.utils as utils
+import controllers.tests.controller_server.host_definer.utils.test_utils as test_utils
 import controllers.tests.controller_server.host_definer.settings as settings
-from controllers.servers.host_definer.types import DefineHostResponse
-import controllers.servers.host_definer.messages as messages
 from controllers.tests.controller_server.host_definer.common import BaseSetUp
+from controllers.servers.host_definer.types import DefineHostResponse
+from controllers.servers.host_definer.watcher.csi_node_watcher import CsiNodeWatcher
 
 
-class TestAddInitialCsiNodes(BaseSetUp):
-    def test_is_host_can_be_defined_not_called_when_there_is_csi_node_but_not_with_ibm_block_provider(self):
-        self.csi_node_watcher._is_host_can_be_defined = Mock()
-        self.csi_node_watcher.csi_nodes_api.get.return_value = self.fake_k8s_csi_nodes_without_ibm_block
+class CsiNodeWatcherBase(BaseSetUp):
+    def setUp(self):
+        super().setUp()
+        self.csi_node_watcher = test_utils.get_class_mock(CsiNodeWatcher)
+        self.mock_nodes_on_csi_node_watcher = test_utils.patch_nodes_global_variable(settings.CSI_NODE_WATCHER_PATH)
+        self.fake_updated_daemon_set = test_utils.get_fake_k8s_daemon_set_items(1, 1)
+        self.fake_not_updated_daemon_set = test_utils.get_fake_k8s_daemon_set_items(0, 1)
+        self.fake_deleted_daemon_set = test_utils.get_fake_k8s_daemon_set_items(0, 0)
+        self.mock_secret_ids_on_csi_node_watcher = test_utils.patch_secret_ids_global_variable(
```

Comment:
IMO no need for `.mock_` or `.fake_` (it's implied by `get_fake` and `patch`)

Reply:
done